esp/scm/ws_access.ecm

```diff
@@ -572,6 +572,163 @@ ESPresponse ResourcePermissionsResponse
     ESParray<ESPstruct ResourcePermission, Permission> Permissions;
 };
 
+
+ESPstruct View
+{
+    string viewname;
+    string description;
+};
+
+ESPrequest QueryViewsRequest
+{
+};
+
+ESPresponse QueryViewsResponse
+{
+    bool success;
+    string result;
+    ESParray<ESPstruct View> views;
+};
+
+ESPrequest AddViewRequest
+{
+    string viewname;
+    string description;
+};
+
+ESPresponse AddViewResponse
+{
+    bool success;
+    string result;
+};
+
+ESPrequest DeleteViewRequest
+{
+    string viewname;
+};
+
+Espresponse DeleteViewResponse
+{
+    bool success;
+    string result;
+};
+
+ESPenum LogicalFileType : string
+{
+    File("File"),
+    Index("Index"),
+};
+
+ESPstruct ViewColumn
+{
+    string viewname;
+    string filename;
+    ESPenum LogicalFileType filetype;
+    bool   isSuperfile;
```

Comment:
This doesn't appear to be set anywhere.  Is it left over from a cut/paste?  Same for filetype.